discovery/gossiper_test.go

```diff
@@ -3602,12 +3602,15 @@ out:
 	// Now that all of our channels are loaded, we'll attempt to update the
 	// policy of all of them.
 	const newTimeLockDelta = 100
+	const newMaxHtlc = (1 << 63) - 1
```

Comment:
Shouldn't this be rejected by our logic since it's greater than the size of the channels themselves?

Reply:
It isn't rejected because the capacity of the test channels is set to zero. This appears to be an exception for light clients. But I removed this code completely, as the gossiper isn't involved anymore with specifics of the policy update.